### PR TITLE
release 0.9.0

### PR DESCRIPTION
**Pull request by audrium**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-- Adds validation of REANA specification to ``start_workflow`` endpoint.
-- Adds interactive sessions out-of-sync check to ``reana-admin check-workflows`` command.
-- Adds workspace retention rules validation to ``get_workspace_retention_rules`` function.
+- Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
+- Adds new ``get_workflow_retention_rules`` endpoint that allows to retrieve the workspace file retention rules of a workflow.
 - Adds ``queue-consume`` command that can be used by REANA administrators to remove specific messages from the queue.
 - Adds configuration environment variable to set an API rate limit for slow endpoints (``REANA_RATELIMIT_SLOW``).
-- Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
-- Adds the possibility to fetch the workflow specification from a remote URL.
+- Adds the possibility to fetch the workflow specification from a remote URL using the new ``/api/launch`` endpoint.
```
Comment (mdonadoni) [inline CHANGES.rst:11]: Maybe we can remove this, as we already have one line for the ``/api/launch`` endpoint?

File: CHANGES.rst [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
-- Adds `retention-rules-apply` command that can be used to apply pending retention rules.
-- Adds `retention-rules-extend` command that can be used to extend the duration of active retentions rules.
-- Changes workflow create endpoint to populate workspace retention rules for the workflow.
-- Changes workflow list endpoint to return workspace retention rules for the workflow.
-- Changes workflow start endpoint to disallow restarting a workflow when retention rules are pending.
+- Adds ``retention-rules-apply`` command that can be used by REANA administrators to apply pending retention rules.
+- Adds ``retention-rules-extend`` command that can be used by REANA administrators to extend the duration of active retentions rules.
+- Changes OpenAPI specification to include missing response schema elements and some other small enhancements.
+- Changes ``/api/info`` endpoint to include kubernetes maximum memory limit, kubernetes default memory limit and the maximum workspace retention period to the response.
+- Changes ``start_workflow`` endpoint to perform REANA specification validation.
```
Comment (mdonadoni) [inline CHANGES.rst:17]:
```suggestion
- Changes ``start_workflow`` endpoint to validate the REANA specification of the workflow.
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+- Changes ``check-workflows`` command that can be used by REANA administrators to include a check for out-of-sync interactive sessions.
```
Comment (mdonadoni) [inline CHANGES.rst:18]: This should probably be `Adds ...`, as the command was added in 0.9.0 (https://github.com/reanahub/reana-server/commit/495a42251d9d19d20f919dc95583a858f06e004b). It should also mention that the command also detects out-of-sync workflows.

File: CHANGES.rst [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
+- Changes ``create_workflow`` endpoint to populate workspace retention rules for the workflow.
+- Changes ``get_workflows`` endpoint to return workspace retention rules for the workflow.
```
Comment (mdonadoni) [inline CHANGES.rst:20]: This should be removed (see https://github.com/reanahub/reana-server/commit/670dbc2e4a7be9a1623b9aa1cf0212f8e8f274c2)

File: CHANGES.rst [hunk 5]
```diff
@@ -1,30 +1,31 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
+Version 0.9.0 (2022-12-14)
```
[the remark this answers is not in the document]
Comment (audrium) [reply CHANGES.rst:4]: I think it's very minor fix and it does not look worth mentioning

**Review by mdonadoni — APPROVED** [review]

**Pull request merged**